### PR TITLE
Fix - 119 - reorder projects page

### PR DESCRIPTION
**Pull request by juanan0309**
# What does this PR do?

This PR:
- Reorders the info from the Project page as per the issue
- Do a small refactor to separate the section code in a separate component to increase readability
- Fix a small issue from the head title

# Where should the reviewer start?

In any Projects page

# How should this be manually tested?

- Go to a Project page and validate that the word `milkshake` is no longer present in the tab
- The information should appear in the correct order

# What are the relevant tickets?

#119 

# Screenshots

image

image

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [X] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: app/routes/projects/$projectId/index.tsx [hunk 1]
```diff
@@ -131,7 +130,7 @@ export const meta: TypedMetaFunction<typeof loader> = ({ data, params }) => {
 
   const { project } = data;
   return {
-    title: `${project?.name} milkshake`,
+    title: project?.name,
```
Comment (jackbravo) [inline app/routes/projects/$projectId/index.tsx:133]: 😂

**Review by jackbravo — COMMENTED** [review]
I see this as good also. Only nitpick I have is that I don't like creating a new component for something that is only used in 1 place, and is not really reused.

I rather see the code there even if the file is bigger, but at least you can see everything in one place and rearrange also everything in one place. This is just my personal preference. So feel free to merge it as it is or to modify it.

**Comment by juanan0309** [comment]
> I see this as good also. Only nitpick I have is that I don't like creating a new component for something that is only used in 1 place, and is not really reused.
> 
> I rather see the code there even if the file is bigger, but at least you can see everything in one place and rearrange also everything in one place. This is just my personal preference. So feel free to merge it as it is or to modify it.

Hi @jackbravo!
I totally see your point, I am the kind of person who is stressed with a lot of JSX though haha, but if you prefer I can turn it back and leave it all in the projects page, not a problem at all. @martinrobled0, @LuisTejedaS, @maubarraganwize any comments about this? Let's make it a group decision

**Comment by LuisTejedaS** [comment]
> > I see this as good also. Only nitpick I have is that I don't like creating a new component for something that is only used in 1 place, and is not really reused.
> > I rather see the code there even if the file is bigger, but at least you can see everything in one place and rearrange also everything in one place. This is just my personal preference. So feel free to merge it as it is or to modify it.
> 
> Hi @jackbravo! I totally see your point, I am the kind of person who is stressed with a lot of JSX though haha, but if you prefer I can turn it back and leave it all in the projects page, not a problem at all. @martinrobled0, @LuisTejedaS, @maubarraganwize any comments about this? Let's make it a group decision

Hi @juanan0309, what do you think would be the advantages of having a new component? I think that maybe this component could be tested in isolation, or of course be reused somewhere else. ATM we are not using this component in other parts of the code and we are not testing too much :). But i don't see this new component as a thing we should avoid.  I just want to know if we have other advantages in our minds.

**Comment by martinrobled0** [comment]
> > I see this as good also. Only nitpick I have is that I don't like creating a new component for something that is only used in 1 place, and is not really reused.
> > I rather see the code there even if the file is bigger, but at least you can see everything in one place and rearrange also everything in one place. This is just my personal preference. So feel free to merge it as it is or to modify it.
> 
> Hi @jackbravo! I totally see your point, I am the kind of person who is stressed with a lot of JSX though haha, but if you prefer I can turn it back and leave it all in the projects page, not a problem at all. @martinrobled0, @LuisTejedaS, @maubarraganwize any comments about this? Let's make it a group decision

Well, in my personal opinion I'm also that kind of person that doesn't like all the code in the same document. This approach could help when we added the github information, in order to separate the two kind of information. But yeah @jackbravo has a point.

**Comment by juanan150** [comment]
Thank you all for your comments!
Regarding your questions, it is just a personal preferences, this is a tricky topic with no right answers, the only advantage I could see is that presentational components are easier to test so this can be tested easier, but as we don't have tests for this page (yet) it is not improving anything right now. And that's it haha. I think I will rollback that change and send only the requested changes without the composition, and maybe we can talk about this later to see if we agree on doing more composition even for components that will be used once 👍

**Pull request merged**